beacon-chain/sync/rpc_send_request.go

```diff
@@ -787,3 +815,14 @@ func downscorePeer(p2p p2p.P2P, peerID peer.ID, reason string, fields ...logrus.
 	newScore := p2p.Peers().Scorers().BadResponsesScorer().Increment(peerID)
 	log.WithFields(logrus.Fields{"peerID": peerID, "reason": reason, "newScore": newScore}).Debug("Downscore peer")
 }
+
+func DataColumnSidecarsByRangeRequest(columns []uint64, start, end primitives.Slot) *ethpb.DataColumnSidecarsByRangeRequest {
+	sort.Slice(columns, func(i, j int) bool {
+		return columns[i] < columns[j]
+	})
```

Comment:
Do we really need to sorts `columns`? Is it in the spec?
(If we do need, the following suggestion does the same with less code.)

```suggestion
    slices.Sort(columns)
```

Reply:
It was nice when developing but not necessary in release.